src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -24,6 +31,18 @@
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        return movieDao.getAll();
+    }
+
+    @Override
+    public Boolean update(Movie movie) {
```

Comment:
Consider changing the return type of the `update` method to `void` if the operation's success can be determined by exceptions or other means. Returning a `Boolean` might not be necessary.